### ISSUE TITLE
Is it possible to make null driver write image to a external device, hdmi, for example

### ISSUE THREAD

**Issue by alex9099** [ValveSoftware/SteamVR-for-Linux#59]
Hi, i finally got past the 306/307 error (i used nvidia-xrun on my optimus laptop) and now i'm wondering if it is possible to show the game image on an external display, i cant find reference to that anywhere :(

**Comment by her001** [comment]
I think this is outside the scope of SteamVR. You could try one of the many X11 manipulation tools, like possibly `xdotool`, to move the window onto that display.

**Comment by alex9099** [comment]
no, i mean, having some kind of direct mode :) not really sure how that works, but i think it is faster to render or something like that
